app/views/layouts/application.html.erb

```diff
@@ -2,44 +2,51 @@
 <html>
 <head>
   <title>Places</title>
+  <meta charset='utf-8'>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <%= javascript_include_tag :application %>
   <%= stylesheet_link_tag 'application', media: 'all' %>
-  <%= csrf_meta_tags %>
+  <%= stylesheet_link_tag 'index', media: 'all' %>
+  <%= javascript_include_tag "vendor/modernizr" %>
+  <%= csrf_meta_tag %>
+  <%= favicon_link_tag "p.ico" %>
+  <title>Inventory</title>
 </head>
 <body>
-  
-  <nav class="top-bar">
-    <ul class="title-area">
-      <li class="name">
-        <h1><a href="/">Places</a></h1>
-      </li>
+
+  <div id='cssmenu'>
+    <ul>
+      <li><a><%= link_to "Home", root_path %></a></li>
+      <%- if current_user -%>
+      <li><a><%= link_to 'New', new_listing_path %></a> </li>
+      <li><a><%= link_to 'Profile', user_path(current_user) %></a></li>
+      <li><a><%= link_to 'Account', edit_user_registration_path%></a></li>
+      <li><a><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></a></li>
+      <%- else -%>
+      <li><a><%= link_to 'Sign Up', new_user_registration_path, id:"a" %></a></li>
+      <li><a><%= link_to 'Sign In', new_user_session_path %></a></li>
+      <%- end -%>
     </ul>
+  </div>
 
-    <section class="top-bar-section">
-      <ul class="right">
-        <%- if current_user -%>
-          <li><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></li>
-        <%- else -%>
-          <li><%= link_to 'Sign Up', new_user_registration_path %></li>
-          <li><%= link_to 'Sign In', new_user_session_path %></li>
-        <%- end -%>
-      </ul>
-    </section>
-  </nav>
+  <div id="parallax-world">
 
-  <%- flash.each do |key, value| -%>
-    <div class="flash flash-<%= key %>">
-      <%= value %>
-    </div>
-  <%- end -%>
+  <div class="block">
+    <%= render 'layouts/flash_notice' %>
+  </div>
 
-  <%= yield %>
+    <%= yield %>
 
-  <%= javascript_include_tag 'application' %>
-  <%= javascript_tag do %>
+    <%= javascript_tag do %>
     $(function(){
       $(document).foundation();
     });
```

Comment:
```
<%= javascript_tag do %>
    $(function(){
      $(document).foundation();
    });
    <% end %>
```
This block is in your application.js file, so you can delete it here. This is an issue with the make_it_so gem.